apps/mobile-wallet/src/components/layout/ScrollScreen.tsx

```diff
@@ -88,7 +88,7 @@ const ScrollScreen = ({
         alwaysBounceVertical={true}
         onScroll={screenScrollHandler}
         onScrollEndDrag={scrollEndHandler}
-        style={{ overflow: SCREEN_OVERFLOW, paddingTop: headerOptions ? 15 : 0 }}
+        style={{ overflow: SCREEN_OVERFLOW }}
```

Comment:
@mvaivre this fixes the bottom padding issues on Android. I couldn't find any side-effects of this change. Maybe you remember better why we had this? I checked both iOS and Android

Reply:
Nope I can't remember. I don't find any issues on iOS either sooo 🤷